go/ql/lib/semmle/go/dataflow/internal/DataFlowNodes.qll

```diff
@@ -21,9 +21,14 @@ module Private {
   DataFlowCallable nodeGetEnclosingCallable(Node n) {
     result.asCallable() = n.getEnclosingCallable()
     or
-    (n = MkInstructionNode(_) or n = MkSsaNode(_) or n = MkGlobalFunctionNode(_)) and
+    not n instanceof FlowSummaryNode and
```

Comment:
Does this now (intentionally) include `MkImplicitVarargsSlice`?

Reply:
Yes.